File: IT/Interfacce grafiche/Esercizi/Java/Esercizio9.java
/*
file Esercizio9.java
passare un parametro da linea di comando, per esempio
javac Esercizio9.java
java Esercizio9.java 'Ciao, mondo!' (Attenzione ai singoli apici) 
*/

import javax.swing.JFrame;
import javax.swing.JLabel;

public class Esercizio9 {
	public static void main(String [] args) {
		String arg = args.length > 0 ? args[0] : "La tua stringa";

		JFrame frame = new JFrame();
		JLabel label = new JLabel(arg);

		frame.add(label);
		frame.pack();

		frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
		frame.setVisible(true);
	}
}

File: IT/2/Interfacce grafiche/Esercizi/Python/esercizio9.py
# file esercizio9.py
# passare un parametro da linea di comando, per esempio
# python esercizio9.py 'Ciao, mondo!' (Attenzione ai singoli apici) 

import sys

try:
	from tkinter import Frame, Label
except ImportError:
	from Tkinter import Frame, Label

arg = sys.argv[1] if len(sys.argv) > 1 else "La tua stringa"

frame = Frame()

label = Label(frame, text=arg)
label.pack()

frame.pack()
frame.mainloop()

